app/models/site.rb

```diff
@@ -24,4 +28,11 @@ def instance
       first_or_create
     end
   end
+
+  private
+
+    def update_banner_config
+      # If a site banner_image uploaded, then update Sufia's banner_image to use it
+      Sufia.config.banner_image = banner_image? ? banner_image.url : nil
```

Comment:
Crud.  You all are correct, `Sufia.config` is completely in-memory and therefore incorrect. If I simply restart the rails server this updated `banner_image` is not persisted.  

I need to find a different way of persisting this change, or I need to override the default Sufia landing page to use `@site.banner_image.url` by default.  Hints are welcome, but I'll also do some digging here.